sendgrid/sendgrid.py

```diff
@@ -73,6 +77,27 @@ def _get_default_headers(self):
     def reset_request_headers(self):
         self.client.request_headers = self._get_default_headers()
 
+    def make_request(self, request_obj, query_params=None, method="get", request_body=None):

```

Comment:
I believe this code should be incorporated here: https://github.com/sendgrid/python-http-client/blob/master/python_http_client/client.py#L158 and be configured in the constructor here: https://github.com/sendgrid/sendgrid-python/blob/master/sendgrid/sendgrid.py#L35